.travis.yml

```diff
@@ -6,11 +6,13 @@ scala:
 jobs:
   include:
     - stage: publishLocal
-      script: sbt ++$TRAVIS_SCALA_VERSION publishLocal
+      script: sbt ++$TRAVIS_SCALA_VERSION publishLocal publishM2
```

Comment:
Hard to say how this pans out... also it doesn't seem like it's being picked up - which is strange, does that mean `publishLocal` didn't actually have any effect either?

Reply:
Ahh, from https://docs.travis-ci.com/user/build-stages/#Data-persistence-between-stages-and-jobs:

Reply:
Right - so we could perhaps use the pattern from https://docs.travis-ci.com/user/build-stages/warm-cache/ to warm up the ivy cache (when we configure it as a cache) but not rely on it for the pipeline. Makes sense. For now remove the publishLocal stage then?

Reply:
Yea. Will remove the stage.